src/blocks/recommendations/class-recommendations-block.php

```diff
@@ -75,15 +75,15 @@ public static function render_callback( array $attributes ): string {
 		wp_enqueue_script( 'wp-parsely-recommendations-view-script' );
 		ob_start();
 		?>
-<section
+		<section
```

Comment:
I think this will inject unneeded tabs into the markup. This is probably not an issue, but wanted to pinpoint it.

Reply:
It was causing issue while folding/unfolding the functions in the code editor. Let me know if we need to revert this otherwise I think this one is better.